src/Data/Table/Base.agda

```diff
@@ -6,46 +6,59 @@
 
 module Data.Table.Base where
 
+open import Data.Nat
 open import Data.Fin
+open import Data.Product using (_×_ ; _,_)
 open import Data.List as List using (List)
-open import Data.Nat
 open import Function using (_∘_)
 
 record Table {a} (A : Set a) n : Set a where
   constructor tabulate
-  field
-    lookup : Fin n → A
-
+  field lookup : Fin n → A
 open Table public
 
-map : ∀ {n a b} {A : Set a} {B : Set b} → (A → B) → Table A n → Table B n
-map f t = tabulate (f ∘ lookup t)
-
+module _ {a} {A : Set a} {n : ℕ} where
```

Comment:
I would prefer the `{n}` parameter to come first, as that's the one that needs to be specified most frequently.

Reply:
I thought the record wrapper meant this was nigh never needed.
(in contrast to, say, `Data.Product.N-ary`'s `n` which is all over the place)

Reply:
I thought I remembered somewhere in my code where it was necessary, but now I can't find it. You're probably right.

Reply:
I think the order is good as is, we should give them in the same order provided to the `Table` type. I think doing otherwise is asking for confusion!